scripts/bygg

```diff
@@ -12,17 +12,50 @@ SCRIPT_FOLDER=$(dirname "$SCRIPTNAME")
 PLATFORM_FOLDER=$(realpath "$(dirname "$SCRIPT_FOLDER")")
 
 # Parse options or show usage which exits app
-usage()
-{
-  echo "$0 - build bitbake recipes/images using docker"
-  echo "Usage: $(basename "$0" ) [options] build-command-to-run"
-  echo "options(: means required argument, :: means optional argument)"
-  for option in $(echo "$LONGOPTIONS" | tr , \\n); do
-    echo "--$option"
-  done
+usage() {
+
+  cat << EOF
+$0 - build bitbake recipes/images using docker
+Usage: $(basename "$0") [options] build-command-to-run
+options(: means required argument, :: means optional argument)
+
+OPTIONS:
+  -b, --build-folder     Optional, specify the folder for the build output. Default is 'build' directory.
+  -c, --delete-conf      Optional, delete existing configuration files in build-folder. Recomended to use if built diffrent machines and multiple machines.
+  -d, --distro           Optional, specify the target poky distro. Example are poky and fslc-wayland. Default is poky.
+  -f, --manifest-file    Required, specify the manifest file to use. See list of manifest files in this repo. Example kirkstone.xml.
+  -g, --skip-docker-pull Optional,  Skip update the docker container.
+  -h, --help             Display this help message and exit
+  -i, --image            Specify the bsp image to use for the build.
+                         Can be combine with recipe option or skiped if 'recipe' option is used.
```

Comment:
What does it mean that it can be combined with a recipe option or skipped if the same option is use. Do you mean to say that it is a required argument if no recipe option is used?

Also, spelling:

combine -> combined
skiped -> skipped